src/drafty.js

```diff
@@ -597,28 +599,61 @@ Drafty.parse = function(content) {
     result.txt = blx[0].txt;
     result.fmt = (blx[0].fmt || []).concat(blx[0].ent || []);
 
+    if (result.fmt.length) {
+      const segments = segmenter.segment(result.txt);
+      for (const ele of result.fmt) {
+        ({
```

Comment:
You use `const {` everywhere else. I suggest to choose one and stick to it: either `const {` or `({`. I like `const {` more because it's more readable.

Reply:
We cannot use `const {` notation here because no new variable is declared. We are assigning the value directly to field of `ele` object. 
If it is required that we use `const { ` notation, something like this would work

```
const {
    at: correctAt,
    len: correctLen
  } = getCorrectLengthsWhenTreatedAsGrapheme(ele, segments, result.txt);
ele.at = correctAt;
ele.len = correctLen;
```